### ISSUE TITLE
Support new setting Keywords Tags in RF 5.1

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#641]
From https://github.com/robotframework/robotframework/issues/4373

This require:
1) updating our visitors (so tags from Keywords Tags are visited and also setting related rules like naming ones does work on new setting)
2) refactoring some of our tag rules

**Comment by mnojek** [comment]
@bhirsz You merged one PR #741 that was related to this. Does it mean this issue can be closed or more work is required here?

**Comment by bhirsz** [comment]
Most likely not, namely we are probably missing rules like:

empty
```
*** Settings ***
Keyword Tags
```

duplicated
```
*** Settings ***
Keyword Tags  tags
Keyword Tags  tags2
```

Also something similar to ``could-be-test-tags``, ``tag-already-set-in-test-tags`` but for keywords
